src/app/components/forms/form/helper/terra-form-field.helper.ts

```diff
@@ -241,4 +243,40 @@ export class TerraFormFieldHelper
 
         return [min || 0, max || Infinity];
     }
+
+    /**
+     * Parses recursively all defaultValues from a formField and it's children.
+     * @param field
+     */
+    public static parseDefaultValueFromFormField(field:TerraFormFieldInterface):any
```

Comment:
Ich denke `parseDefaultValue` genügt, da wir ja schon im TerraFormFieldHelper sind ;-)
```suggestion
    public static parseDefaultValue(field:TerraFormFieldInterface):any
```